src/python/pants/reporting/streaming_workunit_handler.py

```diff
@@ -39,7 +39,7 @@ def __init__(
         scheduler: Any,
         callbacks: Iterable[Callable],
         report_interval_seconds: float,
-        max_workunit_verbosity: LogLevel = LogLevel.INFO,
+        max_workunit_verbosity: LogLevel = LogLevel.DEBUG,
```

Comment:
I'd include a big blurb here explaining why we're doing this, and indicating that it should be configurable by the plugin.